app/scripts/views/mixins/external-links-mixin.js

```diff
@@ -19,7 +19,8 @@ define(function (require, exports, module) {
           var $el = $(el);
           $el
             .addClass('visible-url')
-            .attr('data-visible-url', $el.attr('href'));
+            .attr('data-visible-url', $el.attr('href'))
+            .attr('rel','noopener');
```

Comment:
Can you change `noopener` to `noopener noreferrer` as per [this MDN link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) Firefox doesn't yet support `noopener`, but it does `noreferrer`.

Can you also add a unit test for this?